src/stories/visualisation/LoadMoreTable.tsx

```diff
@@ -0,0 +1,42 @@
+import { BoxProps, Button, HStack } from "@chakra-ui/react";
+import {Table} from "./Table"
+import React from "react";
+
+export interface TableProps extends Omit<BoxProps, "onClick"> {
+  /** Table data */
+  data: Record<string, any>[] | null;
+  /** Table headers and mapping to record keys */
+  headers: { key: string; label: string }[];
+  /** Callback when row is clicked */
+  onRowClick?: (item: Record<string, any>, index: number) => void;
+  /** Label to be used when displaying "no data available" message */
+  label?: string;
+  /** Styling variant to use for the rows */
+  rowVariant?: string;
+  /** feed click behaviour into Load More button */
```

Comment:
You could extend `TableProps` from the component library instead, and avoid duplicating these properties

```suggestion
export interface LoadMoreTableProps extends TableProps {
  /** Load more callback */
```